src/Common.cpp

```diff
@@ -566,8 +566,19 @@ bool RenameFile(const char *oldf, const char *newf)
 #if defined(TUNDRA_UNIX)
     return 0 == rename(oldf, newf);
 #else
-    return FALSE != MoveFileExA(oldf, newf, MOVEFILE_REPLACE_EXISTING);
+    return FALSE != MoveFileExW(ToWideString(oldf).c_str(), ToWideString(newf).c_str(), MOVEFILE_REPLACE_EXISTING);
 #endif
 }
 
+FILE* OpenFile(const char* filename, const char* mode)
+{
+#if defined(TUNDRA_WIN32)
+    FILE* file;
+    if (_wfopen_s(&file, ToWideString(filename).c_str(), ToWideString(mode).c_str()) != 0)
+        CroakErrno("Unable to open file \"%s\"", filename);
```

Comment:
I think it is a little unexpected that windows dies on error here, while other platforms just return a null pointer in case of failure. This should either be documented or (preferably) made consistent.

Reply:
Sounds good. It looks like most callers of `OpenFile` check the FILE pointer and handle those cases. They don't all print errors the same way, so it's probably useful to print the OS error message at this point. I'll change to `PrintErrno` instead of `CroakErrno`.